images/api/src/Auth.js

```diff
@@ -33,7 +33,12 @@ class Auth {
     })
 
     app.get('/me', async (req, res, next) => {
+      res.send(400, { message: 'please include path'});
+    })
+
+    app.get('/me/:path', async (req, res, next) => {
```

Comment:
Request url is `me?path=%2Fadmin%2Fscheme%2Fcb0b78c0-ec8d-11e7-98a8-ed00b3f17ace`, als query-parameter, niet als nieuwe section